gh-clean-notifications

```diff
@@ -1,7 +1,6 @@
 #!/usr/bin/env bash
 set -e
 
-
```

Comment:
Sorry about that, I only realized that this slipped into the commit. Please let me know if you want me to drop that hunk.

Reply:
Nope, that's all good can be left with that line removal.